### PR TITLE
Added Currencty EUR/CAD

### PR DESCRIPTION
**Pull request by poetaster**
I'm currently watching Euro vs. Canadian dollar. How can you tell :)

Removed the strip invocation in the spec since it wouldn't build for me
(virtual box). I'm on the 3.7 SDK on an old ubuntu 18. Maybe my fault?

Build target was 4.3.

### CODE REVIEW

File: rpm/harbour-watchlist.spec [hunk 1]
```diff
@@ -44,8 +44,8 @@ export VERSION_NUMBER=%{version}
 make %{?_smp_mflags}
 
 # >> build post
-echo "Strip the binary!"
```
Comment (andywuest) [inline rpm/harbour-watchlist.spec:47]: please revert this change - i want to keep that for the releases to have a small binary - i also do remove that when developing locally.

**Review by andywuest — COMMENTED** [review]
Everything else looks good :-)

File: rpm/harbour-watchlist.spec (hunk 1 above)
Comment (poetaster) [reply rpm/harbour-watchlist.spec]: Ok. Just did. I'll submit that pull. I'll try to work out why it doesn't work for me. It's kind of common. :) Thanks!

**Review by andywuest — APPROVED** [review]
Looks good now :-) Thanks.

**Comment by poetaster** [comment]
Thanks for the merge! I think, however that I may have missed something? The EUR/USD rates update to reflect current values, but the EUR/CAD values do not? I wonder if I missed somethinng?

**Comment by poetaster** [comment]
Ah, just thanks for the merge. I'm doing a bit of testing and thinking about extending your currency functionality! Thanks again.
